mimic/test/test_resource.py

```diff
@@ -37,7 +37,7 @@ def one_api(testCase, core):
 
     :return: 2-tuple of ``region`` and ``service_id``
     """
-    service_id, api = core._uuid_to_api.items()[0]
+    service_id, api = next(iter(core._uuid_to_api.items()))
```

Comment:
yeah this is a good idiom to know; you get a more sensible error, and you don't have to reify everything into a list before taking a value from it.